requirements.json

```diff
@@ -61,6 +61,19 @@
     "sha256sum": "812c135ce00b570e7240e1d4fbc0b4e2a51928e97589ae9cf7aa08b0156ef6b4",
     "version": "10.4.0.GA"
   },
+  "jbosseap": {
+    "name": "Jboss Enterprise Application Platform",
+    "description": "",
+    "vendor": "Red Hat, Inc.",
+    "modulePath": "model/jbosseap",
+    "bundle": "yes",
+    "targetFolderName": "jbosseap",
+    "dmUrl": "https://developers.redhat.com/download-manager/jdf/file/jboss-eap-7.1.0.Alpha-installer.jar?workflow=direct",
```

Comment:
Lets use 7.0.0.GA version, because we most likely will need to provide Fuse runtime installation as well and latest Fuse release is not playing well with 7.1.0.Alpha.